src/main/java/com/quiz/repository/CreateTable.java

```diff
@@ -84,10 +81,8 @@ public static void createQuizQuestion(){
         } catch (SQLException e) {
             e.printStackTrace();
         }
-
     }
 
-
     public static void createQuizTable(){
         try(Statement statement = DBConnectorConfig.getConnection().createStatement()){
             String query = "DROP SEQUENCE IF EXISTS quiz_id CASCADE;\n" +
```

Comment:
why you changed the sequence ?

Reply:
that wasn't mine this code was written by the friend before me. i changed it in "quiz service" branch.